clients/csrf/transport.go

```diff
@@ -17,38 +20,55 @@ type Cookies struct {
 }
 
 type Transport struct {
-	Transport http.RoundTripper
-	Csrf      *Csrf
-	Cookies   *Cookies
+	OriginalTransport http.RoundTripper
```

Comment:
Is `OriginalTransport` object hold the actual http client?

Reply:
basically yes

Reply:
Wouldn't `delegate` be appropriate here?